data/debian-distro-info.json

```diff
@@ -129,10 +129,5 @@
     "codename": "Trixie",
     "series": "trixie",
     "created": "2023-08-01"
-  },
-  "v14": {
```

Comment:
removed? 🤔

Reply:
I'll let you guys check the source files. 😉

> Actually, with a bit more digging, this is the source for `debian-info-data` and `ubuntu-info-data`:
> 
> * https://debian.pages.debian.net/distro-info-data/debian.csv
> * https://debian.pages.debian.net/distro-info-data/ubuntu.csv

_Originally posted by @JamieMagee in https://github.com/renovatebot/renovate/issues/14881#issuecomment-1086751248_

Reply:
The removal is correct. I assume they removed it as the August 2025 date was a placeholder. The only news I can find about Debian 14 say it is going to be released in 2027[^1].

[^1]: https://www.phoronix.com/news/Debian-14-Forky